libcaf_core/caf/or_else.test.cpp

```diff
@@ -2,16 +2,19 @@
 // the main distribution directory for license terms and copyright or visit
 // https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
 
-#define CAF_SUITE or_else
+#include "caf/test/test.hpp"
 
-#include "caf/all.hpp"
-
-#include "core-test.hpp"
-
-#define ERROR_HANDLER [&](error& err) { CAF_FAIL(err); }
+#include "caf/actor_system.hpp"
+#include "caf/actor_system_config.hpp"
+#include "caf/event_based_actor.hpp"
+#include "caf/response_handle.hpp"
+#include "caf/scoped_actor.hpp"
 
 using namespace caf;
 
+#define ERROR_HANDLER                                                          \
+  [&](error& err) { test::runnable::current().fail("{}", err); }
+
```

Comment:
I think just defining this lambda in `run_testee` directly and passing it along multiple times would be a much better solution than invoking macros.